src/configuration.js

```diff
@@ -32,30 +35,42 @@ export class Configuration {
     }
   }
 
-  getRules() {
+  getRules(searchDirectory) {
```

Comment:
Rather than accept a `searchDirectory` here, what are your thoughts on having the `constructor` accept a `configDirectory` argument?

My reasoning is that eventually other things will be part of this config file and we'd have to accept a `searchDirectory` for all of them. For example, `.graphqlconfig` supports specifying where your `schema` is located. We could leverage this information in `getSchema()` so that people can just run `graphql-schema-linter` with no options at all.